plugin-jdbc/src/main/java/io/kestra/plugin/jdbc/AbstractJdbcQuery.java

```diff
@@ -128,10 +142,24 @@ public AbstractJdbcQuery.Output run(RunContext runContext) throws Exception {
         }
     }
 
+    private void setFetchTypeFromDeprecatedData() {
```

Comment:
We usually use setters for that which would be used at deserialization time to set the value of the field to the new property. But this also work so changing it is not mandatory